examples/javascript/styles.css

```diff
@@ -0,0 +1,33 @@
+body {
+  text-align: center;
+  background-color: #f0dc4e;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: calc(10px + 2vmin);
+  color: #2e302c;
+}
+
+input {
+  width: 250px;
+}
+
+.link-sdk-modal {
+  width: 460px;
+  height: 840px;
+  z-index: 1;
+  position: absolute;
```

Comment:
z-index should be avoided if possible
`position: fixed` > `position: absolute` in this case